### ISSUE TITLE
Feature Request:

### ISSUE THREAD

**Issue by thewan056** [ShankarBUS/ModernFlyouts#277]
**Is your feature request related to a problem? Please describe:**
Album Art displayed looks terrible.

**Describe the solution you'd like:**
Some sort of image resize algorithm implemented to make it look less terrible. Something simple is fine.

**Describe alternatives you've considered:**

**Additional context:**

**Comment by Samuel12321** [comment]
duplicate of #208

**Comment by thewan056** [comment]
I prefer the way the thumbnail is displayed currently. If there are others that request the thumbnail to fill up the given space via cropping/stretching, then there should be an option to select different thumbnail modes. If you want to stick to just one way instead of complicating it with options, my vote is to keep to what is shown currently and implement image resizing to make it look nicer.

**Comment by Cyberdroid1** [comment]
> I prefer the way the thumbnail is displayed currently. If there are others that request the thumbnail to fill up the given space via cropping/stretching, then there should be an option to select different thumbnail modes. If you want to stick to just one way instead of complicating it with options, my vote is to keep to what is shown currently and implement image resizing to make it look nicer.

I agree. Rather than cutting off information of the image, it is still a good thing to keep it filled with blank space. 
I would prefer black color or transparency instead of grey.

**Comment by ShankarBUS** [comment]
@Cyberdroid1,

We need to consult a UX, UI and Logo designer before making further decisions regarding layout, interface or even our logo.

I don't know how to manage them. Me bRaIn smol. I keep messing things up. Please let me search for a right person.

**Comment by Cyberdroid1** [comment]
> @Cyberdroid1,
> 
> We need to consult a UX, UI and Logo designer before making further decisions regarding layout, interface or even our logo.
> 
> I don't know how to manage them. Me bRaIn smol. I keep messing things up. Please let me search for a right person.

I can try 😁 😂
